### PR TITLE
Change colouring perm perc

### PR DESCRIPTION
**Pull request by mark-edmondson-jones**
Updated colouring of permutation and percentile plots to show actual arm with p-adj colouring

### CODE REVIEW

**Comment by Karpefors** [comment]
It looks like you can not select the coloring method? Will it always be colored by p.adj? 
Preferably you should be able to choose the coloring in the same way as in the basic plot.

**Comment by Karpefors** [comment]
Agree. However, it would be good to be able to select "Terms" in order to create a monochrome tendril if needed. The RR and OR are odder and not so useful.

File: R/plotpercentile.R [hunk 1]
```diff
@@ -8,18 +8,25 @@ plotpercentile <- function(x){
...
 
+  plotdata$cc.10<-pmax(log10(plotdata$p.adj), -3)
+  palette <- tendril_palette()
+  
   #plot title
   Title <- paste(unique(perm.data$Terms), ", from day: ", unique(plotdata$perm.from.day), sep = "")
 
   #line colours
-  colours <- c("#F8766D", "darkgoldenrod1", "#00BFC4")
+#  colours <- c("#F8766D", "grey50", "grey80")  # c("#F8766D", "darkgoldenrod1", "#00BFC4")
```
Comment (stefanoborini) [inline R/plotpercentile.R]: Is this supposed to be commented out? If so, just remove it.

**Comment by stefanoborini** [comment]
Review done. A comment. Merge when done.

**Pull request merged**